src/ripple/app/misc/ValidatorList.h

```diff
@@ -163,8 +225,12 @@ class ValidatorList
     // The master public keys of the current negative UNL
     hash_set<PublicKey> negativeUNL_;
 
-    // Currently supported version of publisher list format
-    static constexpr std::uint32_t requiredListVersion = 1;
+    // Currently supported versions of publisher list format
+    static constexpr std::uint32_t supportedListVersions[]{1, 2};
+    // In the initial release, to prevent potential abuse and attacks, any VL
+    // collection with more than 5 entries will be considered malformed.
+    static constexpr std::size_t MAX_SUPPORTED_BLOBS = 5;
```

Comment:
I figure 2 is probably going to be what we're most likely to see, but I wanted to leave a little wiggle room in case someone comes up with some interesting use cases we haven't considered.